OpenRA.Mods.Common/UtilityCommands/DumpSequenceSheetsCommand.cs

```diff
@@ -63,6 +63,13 @@ void IUtilityCommand.Run(Utility utility, string[] args)
 			sb = sequences.SpriteCache.SheetBuilders[SheetType.BGRA];
 			foreach (var s in sb.AllSheets)
 				s.AsPng().Save($"{count++}.png");
+
+			var cursorProvider = new CursorProvider(modData);
+			using (var cursor = new CursorManager(cursorProvider, modData.Manifest.CursorSheetSize))
```

Comment:
The utility description states it does sequences, we should at least update the description to mention cursors are now included.

Although part of me wonders if a different export command for other misc sheets would be better.

Reply:
updated description

Reply:
The `Desc` attribute hasn't changed - did you push?

Reply:
fixed